controllers/v1/solutions.js

```diff
@@ -281,7 +281,11 @@ module.exports = class Solutions extends Abstract {
     /**
     * @api {post} /project/v1/solutions/addRolesInScope/:solutionId Add roles in solutions
     * @apiVersion 1.0.0
+<<<<<<< HEAD
```

Comment:
@Prajwal17Tunerlabs remove these. Make sure merge doesn't cause any code change

Reply:
got it, removed these unwanted characters